crates/env/src/api.rs

```diff
@@ -700,3 +700,30 @@ where
 pub fn set_code_hash(code_hash: &[u8; 32]) -> Result<()> {
     <EnvInstance as OnInstance>::on_instance(|instance| instance.set_code_hash(code_hash))
 }
+
+/// Tries to trigger a runtime dispatchable, i.e. an extrinsic from a pallet.
+///
+/// `call` (after SCALE encoding) should be decodable to a valid instance of `RuntimeCall` enum.
+///
+/// For more details consult
+/// [chain-side documentation](https://github.com/paritytech/substrate/blob/6f72780b87e41a025722284a52a7d85f41958a82/frame/contracts/src/wasm/runtime.rs#L2397).
+///
+/// # Errors
+///
+/// - If the call cannot be properly decoded on the pallet contracts side.
+/// - If the runtime doesn't allow for the contract unstable feature.
+/// - If the runtime doesn't allow for dispatching this call from a contract.
+///
+/// Note however that `call_runtime` host function is still part of the
+/// [unstable interface](https://github.com/paritytech/substrate/tree/master/frame/contracts#unstable-interfaces)
+/// and thus can be changed anytime.
```

Comment:
```suggestion
/// # Note
///
/// The `call_runtime` host function is still part of `pallet-contracts`' unstable inferface and
/// thus can be changed at anytime.
```

Don't think we really need the link here

Reply:
:white_check_mark: